src/averageAges.js

```diff
@@ -52,11 +74,25 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const mothers = new Map(people.map(item => [item.name, item.born]));
```

Comment:
This is intuitive to use Map, but plain object will be enough, will work faster and has shorter syntax